source/_integrations/unifiprotect.markdown

```diff
@@ -212,6 +213,35 @@ Your main UniFi Protect NVR device also gets a number of diagnostics sensors tha
 * **Disk Health**: Each disk installed in your NVR will have a disk health sensor. These are simple good/bad sensors and the order is not promised to match the order in UniFi OS. Disk model number is provided as a state attribute though to help map sensor to disk.
 * **Utilization and Storage Sensors**: Several other sensors are also added for uptime, hardware utilization, and distribution details of the video on disk.
 
+## Media Source
+
+A media source is provided for your UniFi Protect cameras so you can fetch events and video clips and thumbnails.
```

Comment:
```suggestion
A media source is provided for your UniFi Protect cameras so you can fetch events and video clips, and thumbnails.
```

Reply:
Can you change that suggest to this instead?

> A media source is provided for your UniFi Protect cameras so you can fetch events video clips and event thumbnails.